sdk/test/trace/probability_sampler_test.cc

```diff
@@ -0,0 +1,211 @@
+#include "opentelemetry/sdk/trace/samplers/probability.h"
+#include "src/common/random.h"
+
+#include <gtest/gtest.h>
+#include <cstdlib>
+#include <ctime>
+
+using opentelemetry::sdk::trace::ProbabilitySampler;
+using opentelemetry::sdk::trace::Decision;
+using opentelemetry::sdk::common::Random;
+
+namespace
+{
+	int RunShouldSampleCountDecision(
+		opentelemetry::sdk::trace::Sampler::SpanContext& context,
+		ProbabilitySampler& sampler, int iterations)
+	{
+		int actual_count = 0;
```

Comment:
Please run the `format` job on this file.

Reply:
Done